code/modules/detectivework/tools/rag.dm

```diff
@@ -97,7 +97,7 @@
 		user << "<span class='warning'>The [initial(name)] is dry!</span>"
 	else
 		user.visible_message("\The [user] starts to wipe down [A] with [src]!")
-		reagents.splash(A, 1) //get a small amount of liquid on the thing we're wiping.
+		reagents.trans_to(A, 1) //get a small amount of liquid on the thing we're wiping. This uses trans_to instead of splash as it is a direct appliance
```

Comment:
This should be `reagents.splash()`, as you're wiping it down with the rag and just so happen to be getting some inside/on it by accident. You're not wringing the rag out over it.

Reply:
This isn't directly applying, you're wiping it down and some is incidentally getting into it. Splash is appropriate here.